pkg/embeddedetcd/server.go

```diff
@@ -31,6 +31,9 @@ type Server struct {
 }
 
 func NewServer(config CompletedConfig) *Server {
+	if config.Config == nil {
+		return nil
```

Comment:
shouldn't we handle a nil pointer in kcp.go:L134 then?

Reply:
134 is about the kcp server, not etcd server.